composer.json

```diff
@@ -13,12 +13,16 @@
         {"name": "Johannes Schmitt", "email": "schmittjoh@gmail.com"}
     ],
     "require": {
-        "php": "~5.6|~7.0"
+        "php": "~7.1"
     },
     "require-dev": {
+        "alcaeus/mongo-php-adapter": "^1.1",
         "phpunit/phpunit": "^5.7",
         "predis/predis":   "~1.0"
     },
+    "suggest": {
+        "alcaeus/mongo-php-adapter": "Required to use MongoDB driver"
```

Comment:
@alcaeus indeed, I decided to leave it for a different PR to keep things consistent. Thanks for taking it over ;)